benchmarks/pom.xml

```diff
@@ -118,46 +118,6 @@
         </executions>
       </plugin>
     </plugins>
-    <pluginManagement>
```

Comment:
I think those plugin versions should be left in, they are generated by jmh when setting up the pom